src/routes/__layout.svelte

```diff
@@ -1,28 +1,32 @@
 <script lang="ts">
-  import { onMount } from 'svelte';
+  import type { Writable } from 'svelte/store';
   import IconVideoLibrary from '~icons/ic/twotone-video-library';
   import IconFolderShared from '~icons/ic/twotone-folder-shared';
+  import { src, duration, currentTime, paused, playbackRate, buffered } from '$lib/entities/audio';
   import { FullControls } from '$lib/features/playback-controls';
-  import { audio } from '$lib/entities/audio';
   import { IconButton } from '$lib/shared/ui';
 
-  import '$lib/app.css';
+  // Svelte can't handle its own type conversion with TypeScript
+  const bufferedNative = buffered as unknown as Writable<TimeRanges>;
```

Comment:
But how does this work? The type `TimeRanges` is equivalent to 
```ts
type TimeRanges = {
  length: number;
  start: (i: number) => number;
  end: (i: number) => number;
};
```
while buffered has the type `Array<{start: number; end: number}>`